jira/client.py

```diff
@@ -3945,31 +3952,31 @@ def create_project(
 
         ps_list: List[Dict[str, Any]]
 
-        if not permissionScheme:
+        if permissionScheme is None:
             ps_list = self.permissionschemes()
             for sec in ps_list:
                 if sec["name"] == "Default Permission Scheme":
                     permissionScheme = sec["id"]
-                break
-            if not permissionScheme:
+                    break
```

Comment:
also a bug fix here? seems to be logical to have the break there but again this changes functionality

Reply:
i didnt find any bug reports related to this...

Reply:
Yep I only discovered it when trying to get the tests to work on the Jira Cloud instance!